test/onigumo_cli_test.exs

```diff
@@ -13,11 +13,22 @@ defmodule OnigumoCLITest do
     "-c"
   ]
 
+  @invalid_combinations [
+    ["--help", "-C", "invalid"],
+    ["-h", "--invalid"],
```

Comment:
This is, in fact, a different use case. It tests another branch of code: the one for invalid switches. The other combinations test a combination of a positional argument and a valid switch.

This exact case would work on _master_ too and should be tested. Not in this pull request though.

Reply:
Created #261.